### DISCUSSION TITLE
Time to learn automation !

### DISCUSSION THREAD

**Discussion by BonezUK2020** [community/community#144351] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello All,

As technology has moved on I find that I now need to learn some programming and have decided to start learning Python, are there any good online sources where I could start and are there any sites that i can use to obtain certification that is recognized.

**Comment by kivanc57** [comment]
If you also look for **free** certification check this out: https://www.freecodecamp.org/ 
FreeCodeCamp has various courses, interactive practices and free certifications.

Here is the second option: https://www.codecademy.com/
Codecademy is also very interactive and fruitful. I got many certificates back in the days but comparison to the first, only a couple introductory classes to programming languages are for free and I am not sure if the certification is free of charge though.
